### ISSUE TITLE
[BUG] Cannot get uploads from channel

### ISSUE THREAD

**Issue by jtaxiexpress** [Hexer10/youtube_explode_dart#228]
**Describe the bug**
Video cannot be displayed.
How about you guys?

All videos are retrieved from the channel.
However, we are unable to retrieve them.

I am able to retrieve videos from playlist.
I am also able to retrieve comments.

**Enviroment: (please complete the following information):**
youtube_explode_dart: ^1.12.1
Include here the stacktrace (if applicable). 
[✓] Flutter (Channel stable, 3.3.4, on macOS 13.0 22A380 darwin-arm, locale ja-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.72.2)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

**Comment by prateekmedia** [comment]
@Hexer10 The error seems to be in `yt.channels.getUploadsFromPage` method, I tried others in example.dart and they worked but this one resulted in below error:
```dart
Unhandled exception:
FatalFailureException: Failed to get initial data context.
#0      _InitialData.getContentContext (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart:102:7)
#1      _InitialData._getUploads (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart:60:21) 
#2      _InitialData.uploads (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart:57:43)
#3      _InitialData.uploads (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart)
#4      ChannelUploadPage.uploads (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart:18:55)
#5      ChannelUploadPage.uploads (package:youtube_explode_dart/src/reverse_engineering/pages/channel_upload_page.dart)      
#6      ChannelClient.getUploadsFromPage (package:youtube_explode_dart/src/channels/channel_client.dart:133:14)
<asynchronous suspension>
#7      main (file://youtube_explode_dart/example/example.dart:7:7)
<asynchronous suspension>
```

**Comment by Hexer10** [comment]
I just pushed a commit to fix this issue. Getting the next page is still broken tough

**Comment by jtaxiexpress** [comment]
> I just pushed a commit to fix this issue. Getting the next page is still broken tough

Thanks for the reply, I will try to get it to work.
I respect open source developers like you.

My app relies on that feature and I would be happy if you could address it.

**Comment by Hexer10** [comment]
This is should be fixed in v`1.12.2`.
